.github/workflows/pull-evaluation-tests.yaml

```diff
@@ -10,14 +10,21 @@ run-name: "Evaluation tests"
 
 on:
   pull_request_target:
+    types: [labeled, opened, edited, synchronize, reopened, ready_for_review]
     branches:
       - main
     paths-ignore:
+      - ".github/**"
+      - ".reuse/**"
+      - "LICENSES/**"
+      - "config/**"
+      - "data/**"
       - "docs/**"
-      - "**.md"
+      - "scripts/**"
+      - "tests/**"
```

Comment:
why are you excluding everything under the "tests" folder? The "blackbox" tests are under this folder. I think, if you exclude, the there will not be evaluation tests run if blackbox/evaluation tests are changed.

Reply:
This is the requirement in the the issue: https://github.com/orgs/kyma-project/projects/61/views/1?pane=issue&itemId=74792786&issue=kyma-project%7Ckyma-companion%7C136

"Modify the workflow file to exclude triggers for test folder updates, PR title changes, and .md file modifications"